week-1/implement/dedupe.js

```diff
@@ -1 +1,49 @@
-function dedupe() {}
+//manual check
+// function dedupe(input) {
+
+//     let output = [];
+
+//     for(let i=0; i<input.length; i++){
+//       let duplicate = false;
+
+//       for (let j = 0; j < output.length; j++) {
+//         if (input[i] === output[j]) {
+//           duplicate = true;
+//         }
+//       }
```

Comment:
One thing you can do here is to remove the commented code if you no longer need them! Is not ideal to push comment code unlesss it might be useful.